### PR TITLE
feat(installer): add code signing for Win installer

### PR DESCRIPTION
**Pull request by ctrombley**
This PR adds code signing for the Windows MSI installer during the `release-windows-installer` job.

The PFX binary cert package is protected by the `PFX_PASSWORD` secret that has been added to this repo.

Tested on a Win Server 2019 instance.

### CODE REVIEW

File: installers/Windows/NewRelicCLIInstaller/SignMSI.cmd [hunk 1]
```diff
@@ -1,3 +1 @@
-﻿
-makecert.exe -r -pe -ss MY -sky exchange -n CN="New Relic CLI" CodeSign.cer
-signtool sign /v /s MY /n "New Relic CLI" /t http://timestamp.verisign.com/scripts/timstamp.dll bin\x64\Release\NewRelicCLIInstaller.msi
+"C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64\SignTool.exe" sign /f developer-toolkit.pfx /p "%PFX_PASSWORD%" /t http://timestamp.digicert.com bin\x64\Release\NewRelicCLIInstaller.msi
```
Comment (zlesnr) [inline installers/Windows/NewRelicCLIInstaller/SignMSI.cmd:1]: What is the `http://timestamp.digicert.com` doing here?  Just noticing that its `http` and not `https`.

Comment (ctrombley) [reply installers/Windows/NewRelicCLIInstaller/SignMSI.cmd:1]: https://cheapsslsecurity.com/blog/what-is-time-stamping/

Comment (ctrombley) [reply installers/Windows/NewRelicCLIInstaller/SignMSI.cmd:1]: https://knowledge.digicert.com/generalinformation/INFO4231.html

Comment (zlesnr) [reply installers/Windows/NewRelicCLIInstaller/SignMSI.cmd:1]: Ah I see.

**Pull request merged**